client/src/templates/Challenges/redux/current-challenge-saga.js

```diff
@@ -25,7 +23,9 @@ export function* currentChallengeSaga({ payload: id }) {
       }
     };
     try {
-      yield call(post, update.endpoint, update.payload);
+      // Temporarily removed to reduce calls to database
+      // will need to re-import things at the top
+      // yield call(post, update.endpoint, update.payload);
```

Comment:
Should we not comment out the entire `if` block??

Reply:
I did that, and was getting errors about a saga without a yield - so I would have had to remove the saga I suppose. Wouldn't have been too hard, but with this, it still saves to local storage. I Don't think that does anything though.